public/index.html

```diff
@@ -14,6 +14,9 @@
     <meta property="og:image" content="https://wakamaifondue.com/wf-screenshot-square.jpg" />
     <link rel="icon" type="image/svg+xml" href="<%= BASE_URL %>favicon.svg">
     <title>Wakamai Fondue, the tool that answers the question “what can my font do?”</title>
+    <script defer
```

Comment:
Given that navigating to https://trackless.app reveals virtually nothing about what this system is and searching for it seems nearly impossible because more popular things with these terms seem to come up first, I suggest documenting what this is in an HTML comment next to this script load so people with concerns can review what is being used.

Reply:
Just to give a bit of context: the URL is from a side project of mine for an app for privacy-friendly (and cookieless) analytics, but it's still (very) early days and it doesn't have a website yet.

This is great feedback though and I understand the concerns, so maybe it's a good idea to close this PR or put it on hold until I have some online information about what it is, what it tracks, and what it doesn't track? (I hope to have something soon!) // @RoelN

Reply:
Good point. I know this is a tracker built with the best intentions by a developer I trust, but it's good to clarify this to other users as well. I'll put this PR on hold for now.